web/hooks/use-follows-supabase.ts

```diff
@@ -0,0 +1,59 @@
+import { RealtimeChannel } from '@supabase/realtime-js'
+import { useEffect, useState } from 'react'
+import { db } from 'web/lib/supabase/db'
+import { getContractFollows } from 'web/lib/supabase/follows'
+import { useRealtimeChanges } from 'web/lib/supabase/realtime/use-realtime'
+import { useEffectCheckEquality } from './use-effect-check-equality'
+import { Delete, Insert } from 'common/supabase/realtime'
+import { uniq } from 'lodash'
+
+export function useRealtimeContractFollows(contractId: string | undefined) {
+  const [follows, setFollows] = useState<string[] | undefined | null>(undefined)
+
+  const deletedFollows: Delete<'contract_follows'>[] = useRealtimeChanges(
+    'DELETE',
+    'contract_follows',
+    {
+      k: 'contract_id',
+      v: contractId ?? '',
+    }
+  )
+  const insertedFollows: Insert<'contract_follows'>[] = useRealtimeChanges(
+    'INSERT',
+    'contract_follows',
+    {
+      k: 'contract_id',
+      v: contractId ?? '',
+    }
+  )
```

Comment:
It seems strange to subscribe to these in two separate channels when you could have done `useRealtimeChanges('*', ...)` to pick up both inserts and deletes over one channel.